pkgs/development/python-modules/cPyparsing/default.nix

```diff
@@ -0,0 +1,20 @@
+{lib, buildPythonPackage, fetchPypi, cython}:
+
+buildPythonPackage rec {
+  pname = "cPyparsing";
+  version = "2.4.5.0.1.1";
```

Comment:
What the hell is this version number. I've never seen so many digits.

Reply:
Dunno, that is how it is in upstream.